cmd/configure.ac

```diff
@@ -257,5 +257,22 @@ fi
 fi
 AM_CONDITIONAL([USE_INTERNAL_BPF_HEADERS], [test "x$host_bpf_headers_usable" = "xno"])
 
+AC_DEFINE([MISSING_FIELD_INITIALIZERS_USABLE], [1], [-Wmissing-field-initializers are usable])
+AC_MSG_CHECKING([whether MISSING_FIELD_INITIALIZER warnings are usable])
+AC_COMPILE_IFELSE([
+#include <linux/resource.h>
```

Comment:
Instead of including I would just make structs.

```
struct { int a; int b; } a = { 0 };
struct { const char* a; int b; } b[] = { {.a = ""}, {} };
```